### ISSUE TITLE
Creating an NFT Collection

### ISSUE THREAD

**Issue by Sanchoyzer** [nessshon/tonutils#15]
Hey. I tried to create an nft collection using this [example](https://github.com/nessshon/tonutils/blob/main/examples/nft/deploy_onchain_collection.py), but I got an error:
> aiohttp.client_exceptions.ClientResponseError: 406, message='error code: 0 message: cannot apply external message to current state : Failed to unpack account state', url=URL('https://tonapi.io/v2/blockchain/message')

I used the original code with several changes: `API_KEY`, `MNEMONIC`, `OWNER_ADDRESS`, `IS_TESTNET=False`. `OWNER_ADDRESS` exists, `API_KEY` and `MNEMONIC` are correct. What shall I do for successfully create a collection?

**Comment by nessshon** [comment]
@Sanchoyzer What wallet version are you using in MNEMONIC?

**Comment by nessshon** [comment]
Check if there is a balance on the wallet and if the wallet version is correct

**Comment by Sanchoyzer** [comment]
@nessshon , it's a good idea. I use a wallet created in Tonkeeper, and it has the W5 version. But in the example there was another version -- V4R2. So I changed the version, and WalletV5R1 class works fine in my case. Thanks

But it might be another issue: in the [example](https://github.com/nessshon/tonutils/blob/main/examples/nft/deploy_onchain_collection.py#L37), several social links are used, but I checked via Getgems service, and a deployed collection doesn't have any
